### PR TITLE
Staging PR

### LINKED ISSUE #461: [Locale] Taxonomy points to EN document
Taxonomy cannot be localised because `en` is hardcoded in the url: https://github.com/adobe/theblog/blob/master/scripts/taxonomy.js#L18

### LINKED ISSUE #463: Topic filter skips older posts
Go to https://blog.adobe.com/en/topics/customer-stories.html and filter for _XD_. Only 5 articles show up.

### LINKED ISSUE #465: Consider retiring special localhost handling for index
a while back when we had no `simulator` support for the `content-proxy` `.json` representation we added a the following files as a static copy...

```
query-index-0.json
query-index-256.json
```

while there still may be merit in local caching to speed up local dev, i'd rather rely on browser caching (or possibly some optional `localStorage` trickery of the first few segments of the actual index)

### PR DESCRIPTION
**Pull request by rofe**
- Fix #461
- Fix #463 
- Fix #465
- Card and author image optimization
- Portrait image support

### CODE REVIEW

File: scripts/common.js [hunk 1]
```diff
@@ -330,7 +330,7 @@ export async function fetchArticleIndex(offset) {
   if (index.done) return;
   let indexUrl;
   
-  if (isLocalhost()) {
+  if (isLocalhost() && window.localStorage.getItem('localIndex')) {
```
Comment (kptdobe) [inline scripts/common.js:333]: What is this ?

**Review by kptdobe — APPROVED** [review]

File: scripts/common.js (hunk 1 above)
Comment (rofe) [reply scripts/common.js:333]: By default, local development now uses the live remote index. This local storage property allows to switch to the static local index - if you want that.

Comment (davidnuescheler) [reply scripts/common.js:333]: interesting. i am not sure if the local index is needed altogether, which might let us cleanup the root a little bit. this is not a comment on this particular PR but something that i think we could consider... i'll file an issue.

Comment (rofe) [reply scripts/common.js:333]: I found it increasingly inconvenient to work with the static local index.

Comment (rofe) [reply scripts/common.js]: Removing special handling, see #465

**Pull request merged**